### PR TITLE
Fix several code smells found by SonarCloud

### PR DESCRIPTION
**Pull request by sjinks**

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ function _configure_wp_parsely_specified_version() {
 	$specified = getenv( 'WPVIP_PARSELY_INTEGRATION_PLUGIN_VERSION' );
 	if ( $specified ) {
 		echo '[WP_PARSELY_INTEGRATION] Specifying plugin version: ' . esc_html( $specified ) . "\n";
-		add_filter( 'wpvip_parsely_version', function ( $passed ) use ( $specified ) {
+		add_filter( 'wpvip_parsely_version', function () use ( $specified ) {
```
Comment (sjinks) [inline tests/bootstrap.php:103]: Unused argument

File: tests/test-cache-ttl-manager-rest.php [hunk 2]
```diff
@@ -32,11 +32,9 @@ public function tearDown(): void {
...
 
 	protected function dispatch_request( $method ) {
-		$request             = new \WP_REST_Request( $method, '/tests/v1/endpoint' );
-		$response            = $this->server->dispatch( $request );
-		$dispatched_response = apply_filters( 'rest_post_dispatch', rest_ensure_response( $response ), $this->server, $request );
-
-		return $dispatched_response;
+		$request  = new \WP_REST_Request( $method, '/tests/v1/endpoint' );
+		$response = $this->server->dispatch( $request );
+		return apply_filters( 'rest_post_dispatch', rest_ensure_response( $response ), $this->server, $request );
```
Comment (sjinks) [inline tests/test-cache-ttl-manager-rest.php:37]: Do not create a temporary variable; return the result immediately

File: tests/test-cache-ttl-manager-rest.php [hunk 3]
```diff
@@ -78,22 +76,9 @@ public function test__set_ttl_for_unauthenticated_write_requests( $method ) {
...
-		$response_headers = $response->get_headers();
-
-		$this->assertArrayNotHasKey( 'Cache-Control', $response_headers );
-	}
-
-	/**
 	 * @dataProvider get_rest_write_methods
 	 */
-	public function test__skip_ttl_for_authenticated_write_requests( $method ) {
+	public function test__skip_ttl_for_authenticated_requests( $method ) {
```
Comment (sjinks) [inline tests/test-cache-ttl-manager-rest.php:81]: `test__skip_ttl_for_authenticated_read_requests()` and `test__skip_ttl_for_authenticated_write_requests()` had identical implementation

**Review by jesse-greathouse — APPROVED** [review]
👍

**Pull request merged**